lib/registry.js

```diff
@@ -201,12 +201,24 @@ registry.configureModel = function(ModelCtor, config) {
       'must be an array of objects', modelName);
   }
 
+  // Remote methods
+  if (typeof settings.methods === 'object' && settings.methods !== null) {
+    var methods = settings.methods = settings.methods || {};
+    Object.keys(settings.methods).forEach(function(key) {
+      ModelCtor.remoteMethod(key, settings.methods[key]);
```

Comment:
> Can you please elaborate more why my proposal is impractical for loopback-workspace to implement

The workspace uses keys as identifiers for entities (in the case a method would be an entity).

> `prototype.methodName` is always mapped to isStatic:false, regardless of the value in the config object

I like where you are going with this... but consider what the workspace does. It loads properties, and other `Entities` and also writes them back to files. We would have to store a bit that says "this method was defined using `prototype.methodName`" and recall that during serialization. Of course this is all possible... but probably not worth it to get rid of the annoying `isStatic` property.